app/models/site_setting.rb

```diff
@@ -184,7 +184,7 @@ def self.Upload
   end
 
   def self.whispers_allowed_group_ids
-    if SiteSetting.enable_whispers && SiteSetting.whispers_allowed_groups.present?
+    if SiteSetting.whispers_allowed_groups.present?
```

Comment:
Probably worth doing in _another_ PR/commit but maybe the `.present?` check should be done in the [`_map`](https://github.com/discourse/discourse/blob/f2d0832618dac335113446c035f9612385622084/lib/site_setting_extension.rb) helper? That way, we don't have to check for existence every single time we use that helper.

(cc @martin-brennan)

Reply:
It might be a premature optimisation but I _think_ the `if foo.presence?` test might be faster than doing `.to_s_split("|").map(&:to_i)` 🤔